confgenerator/testdata/invalid/linux/logging-processor_reserved_id_prefix/golden_error

```diff
@@ -1 +1,9 @@
-logging processor id "lib:processor_1" is not allowed because prefix 'lib:' is reserved for pre-defined processors.
\ No newline at end of file
+the agent config file is not valid. detailed error: [9:20] Key: 'Logging.processors[lib:processor_1]' Error:Field validation for 'processors[lib:processor_1]' failed on the 'startsnotwith' tag
```

Comment:
The new message does not mention what prefix is forbidden. Not too bad though as we could add a note to the troubleshooting guide.

Reply:
Improving all of these error messages is a TODO. We can easily add that back and more, but I'm prioritizing getting a stable interface ready for BM first.